acdc-ws/conf/routes

```diff
@@ -1,3 +1,4 @@
 GET     /__status                   controllers.StatusController.status()
+GET     /__metrics                  controllers.MetricController.collect()
```

Comment:
this would make the configuration of `acdc.metrics.endpoint` useless

Reply:
I see, you removed it from application.conf too

Reply:
Yes, Simon and I talked and concluded a configurable metric endpoint would require introducing another router.  We chose a simpler approach to enable/disable metric collection via configuration.